src/controllers/plugins/Plugin.cpp

```diff
@@ -270,5 +273,19 @@ bool Plugin::hasFSPermissionFor(bool write, const QString &path)
     return false;
 }
 
+bool Plugin::hasHTTPPermissionFor(const QUrl &url)
+{
+    // XXX: Older compilers don't have support for std::ranges
+    // NOLINTNEXTLINE(readability-use-anyofallof)
```

Comment:
Can disable the `readability-use-anyofallof` lint in `.clang-tidy` instead

Reply:
Which compiler doesn't support ranges? macOS supports it from Xcode 14.3 (we're on macos-13, so it's Xcode 15), Ubuntu is on GCC 10, which supports ranges too, and MSVC supports them.

Reply:
idk, just reading the comment

Reply:
This code was copy-pasted from the method above created in #5231.

> I don't think GCC 9 implements the ranges proposal (Ubuntu 20.04).
https://github.com/Chatterino/chatterino2/pull/5231#discussion_r1511419810

I'll give it a try and see if it builds.

Reply:
Fixed in [1cca807](https://github.com/Chatterino/chatterino2/pull/5383/commits/1cca8079af4ce9d2228ebd3292f9068b6efaac6e)